benchmarks/TFLite/linux-riscv.cmake

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RV64_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+v"
+  "--riscv-v-vector-bits-min=256"
+  "--riscv-v-fixed-length-vector-lmul-max=8"
+)
+
+set(LINUX_RV32_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv32-pc-linux-elf"
+  "--iree-llvm-target-cpu=generic-rv32"
+  "--iree-llvm-target-abi=ilp32"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+zvl512b,+zve32x"
+  "--riscv-v-vector-bits-min=512"
+  "--riscv-v-fixed-length-vector-lmul-max=8"
+)
+
+# CPU, Dylib-Sync, RV64-Generic, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
```

Comment:
MobileSSD and mobilenet v2 should use the quantize version in https://github.com/google/iree-samples/tree/main/tflitehub.